android/gradle.properties

```diff
@@ -2,4 +2,4 @@ customerio.reactnative.kotlinVersion=1.7.21
 customerio.reactnative.compileSdkVersion=30
 customerio.reactnative.targetSdkVersion=30
 customerio.reactnative.minSdkVersion=21
-customerio.reactnative.cioSDKVersionAndroid=[3.5.0,4.0)
+customerio.reactnative.cioSDKVersionAndroid=[3.5.3,4.0)
```

Comment:
https://github.com/customerio/customerio-android/pull/227 needs to be released first for the changes in this PR. If the version is different than anticipated here, we need to update the range to make sure it works well for our customers.

Reply:
https://github.com/customerio/customerio-android/pull/227 release in [`3.5.3`](https://github.com/customerio/customerio-android/releases/tag/3.5.3) ✅